tests/test_generator.py

```diff
@@ -164,13 +163,12 @@ def test_dir_files(fileutils, test_dir):
         model_path = osp.join(test_dir, "dir_test/dir_test_" + str(i))
         assert osp.isdir(model_path)
         assert osp.isdir(osp.join(model_path, "test_dir_1"))
-        assert osp.isfile(osp.join(model_path, "test.py"))
+        assert osp.isfile(osp.join(model_path, "test.in"))
```

Comment:
Just checking: did you manually make the `in`->`py` change? Just want to make sure that `black`/`isort` is not changing strs out from underneath us.

Reply:
Yes, because `black` was getting crazy about the file (as it was not able to parse it) and there was no way to skip it.